src/documents/en/mobile/import-google.html.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Google import"
+layout: "default"
+category: "mobile"
+menuOrder: 4
+toc: true
+---
+
+# Import your Google data in your Cozy
+
+## Retrieve authentication token.
+
+To import your data from your Google account, you should connect you on your account and retrieve an authentication token. 
```

Comment:
@poupotte s/you should connect you on your account/you must connect to your account/ ?